packages/components/package.json

```diff
@@ -0,0 +1,51 @@
+{
+	"name": "@wordpress/components",
+	"version": "1.0.0-alpha.1",
+	"description": "UI Compoonents for WordPress",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"components"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/components/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"dependencies": {
+		"@wordpress/a11y": "1.0.6",
+		"@wordpress/api-request": "^1.0.0-alpha.1",
+		"@wordpress/deprecated": "^1.0.0-alpha.1",
+		"@wordpress/dom": "^1.0.0-alpha.1",
+		"@wordpress/element": "^1.0.0-alpha.1",
+		"@wordpress/hooks": "1.1.6",
+		"@wordpress/keycodes": "^1.0.0-alpha.1",
+		"@wordpress/i18n": "1.1.0",
+		"@wordpress/is-shallow-equal": "1.0.2",
+		"classnames": "2.2.5",
+		"clipboard": "1.7.1",
+		"dom-scroll-into-view": "1.2.1",
+		"http-build-query": "0.7.0",
+		"lodash": "4.17.5",
+		"memize": "1.0.5",
+		"moment": "2.22.1",
+		"mousetrap": "1.6.2",
+		"react-click-outside": "2.3.1",
+		"react-color": "2.13.4",
+		"react-datepicker": "1.4.1",
+		"rememo": "3.0.0",
```

Comment:
I noticed that not only `classnames` is used in a few packages, but also:
`memize`, `rememo` or `uuid`. We probably should create a vendor bundle and put them together there to optimize code send to the browser.